conan/tools/system/pip_manager.py

```diff
@@ -0,0 +1,64 @@
+import venv
+import platform
+import os
+
+from conan.errors import ConanException
+from conan.tools.env.environment import Environment
+
+
+class Pip:
```

Comment:
```suggestion
class PipEnv:
```

Or something in that line? something that makes explicit in the name that will install in a separate Venv, not the current one.